library/include/rocblas.h

```diff
@@ -13,6 +13,7 @@
 #include "rocblas-auxiliary.h"
 #include "rocblas-export.h"
 #include "rocblas-functions.h"
+#include "rocblas-profile.h"
```

Comment:
The reason I wanted to avoid doing this is because I wanted to keep this API "hidden". Since it is not in the main rocblas-functions, I guess it's good enough?

Reply:
It should probably be public, and not "hidden", but I agree with separating the functions into their own headers.

Reply:
What do others think about keeping it "hidden", or making it a part of the headers included by `rocblas.h`? I favor making it a part of the functions declared when `rocblas.h` is `#include`'d, but I agree with putting it in a separate header than `rocblas-functions.h`.